README.md

```diff
@@ -217,21 +220,30 @@ Step 2:
 | coverage      | 7.2.7      |   Apache Software License (Apache-2.0) |
 
   
-## Roadmap
-
-- [x] ~Make the pages modular using React~
-    - [x] ~Login/SignUp~ 
-    - [x] ~Homepage~
-    - [x] ~Events~
-    - [x] ~Contact Us~
-    - [x] ~Profile~    
-- [x] ~Allow users to enroll in events~
-- [x] ~Generate graphs for user's calories~
-- [x] ~Show consumed food history~
-- [ ] Include UI testing Framework
-- [ ] Allow sharing/exporting user data
-- [ ] Dockerise entire application for easy deployment
-- [ ] TDEE Calculator
+# What's new?
+
+This version enhanced and built on the previous version, making it 10x more interesting and easier to use!
+Here's what we added in this release.
+
+1. Ability to add custom foods and track their calories.
+2. Fully customizable meal creation.
+3. An intuitive and searchable dropdown.
+4. BMI Calculation for your personal profile.
+5. Intelligent automatic calorie burn calculation.
+6. Enhanced UI.
+7. Randomized Exercise of the Day.
+8. Google Sign-in is now an option!
+9. Better progress tracking on the landing page.
+
+## Bug Fixes
+We fixed many bugs, but here are the most notable ones...
+
+1. Sign up / Sign in issues, especially when an existing user tries to sign up again.
+2. The proxy in the config files were wrong, making it difficult to run the website. Proxy issues were fixed.
+3. Testcases were updated and fixed.
+4. Event were not un-enrollable. Now events can be enrolled and unenrolled.
+5. Exercise of the day was static, which defeats the purpose. Fixed, it is now random.
+6. Linechart on the landing page was incorrect. Fixed.
```

Comment:
I would suggest using slightly more constructive wordings instead of "incorrect"/"defeats the purpose".

Reply:
Sure. Please provide some substitutes!

Reply:
Proxy in the config files did not work as expected on all systems. Fixed it to make it run correctly everywhere.
Landing page cards were not responsive to different screen sizes.
Exercise of the day was static, Now truly random